korgw-sample/src/commonMain/kotlin/com/soywiz/korgw/sample/KorgwSample.kt

```diff
@@ -68,144 +53,103 @@ class KCube {
         name = "MY_PROG"
     )
 
-    fun run() = DefaultGameWindow.mainLoop {
-            configure(600, 600, "Multicolored cube", fullscreen = false)
+    var projMat = Matrix3D().setToPerspective(aov, ar, near, far)
+    var viewMat = Matrix3D().setToTranslation(xPos, yPos, zPos)
+    var modlMat = Matrix3D().identity()
+
+    fun run() = Korio {
+        DefaultGameWindow.loop {
+            //TODO: why can't I use width and height vars here ?
```

Comment:
`this@KCube.width` and `this@KCube.height` since
`this: DefaultGameWindow ` is injected as part of the `DefaultGameWindow.loop` DSL.

Reply:
ok